frontend/src/types.ts

```diff
@@ -24,4 +24,5 @@ export interface Chat {
   thread_id: string;
   name: string;
   updated_at: string;
+  metadata: Record<string, unknown> | null;
```

Comment:
Question: Curious, is there any difference between using `unknown` vs `any` here?

Reply:
https://github.com/Microsoft/TypeScript/pull/24439

"unknown is the type-safe counterpart of any. Anything is assignable to unknown, but unknown isn't assignable to anything but itself and any without a type assertion or a control flow based narrowing. Likewise, no operations are permitted on an unknown without first asserting or narrowing to a more specific type."